src/components/Input.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+const Input = ({ label, options, isInput = false, placeholder = "" }) => {
+  return (
+    <div>
+      <label className="block text-black mb-2">{label}</label>
+      {isInput ? (
+        <input
+          type="text"
+          placeholder={placeholder}
+          className="border border-gray-300 p-2 rounded w-64"
+        />
+      ) : (
+        <select className="border border-gray-300 p-2 rounded w-64">
```

Comment:
please keep Input component and Dropdown component separate. As application grows and more options/props gets added it will be easier to scale the atom components

Reply:
done